.gitlab-ci.yml

```diff
@@ -616,9 +616,15 @@ test_backend_integration:
       done
   script:
     - cd integration/backend-tests/
+    - export INTEGRATION_TEST_SUITE=$(integration/extra/release_tool.py --select-test-suite || echo "all")
```

Comment:
ok, `integration/extra/release_tool.py` is computed with `cd integration/backend-tests` as a base, so it's not found. should be moved up?

Reply:
Correct. I'll fix.

Reply:
Fixed